plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestRemarksReportingOraclePoolConnectorSmokeTest.java

```diff
@@ -27,16 +26,16 @@
 import static io.trino.testing.sql.TestTable.randomTableSuffix;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class TestOraclePoolConnectorTest
-        extends BaseOracleConnectorTest
+public class TestRemarksReportingOraclePoolConnectorSmokeTest
```

Comment:
Since we're renaming I'd propose `TestOracleRemarksReportingPoolConnectorSmokeTest`. Having a common prefix (`TestOracle`) makes searching for tests easier and they sort better in the IDE.

Not a strong preference though. 🙂

Reply:
`TestOraclePoolRemarksReportingConnectorSmokeTest`